doc/rfc008-managing-apalache.md

```diff
@@ -0,0 +1,343 @@
+# RFC008: Obtaining and Launching Apalache from Quint
```

Comment:
Could we name the RFCs sequentially starting with #1062?

Using the same numbering for RFCs and ADRs isn't helpful, ime, since they (should) serve pretty different purposes!

```suggestion
# RFC001: Obtaining and Launching Apalache from Quint
```

Reply:
Unless this is actually an ADR? Is seems like it may be.

And what is the difference? I guess maybe more thought is needed here :sweat_smile:

Reply:
I think this has evolved into an ADR. I will revamp it a bit, to record the discussions here.

There is an advantage though in sharing a single counter between RFCs and ADRs: for RFCs that grow up to be an ADR, both documents will share the same index, and their correspondence is easily identifiable.

Reply:
I don't think I agree with the premise that ADRs are an evolutionary stage of RFCs :sweat_smile: IMO, they call for quite different scope and content.

But this is part of a larger process discussion that doesn't effect this PR. I think this is an ADR :+1:

Reply:
I know we have been discussing that multiple times. I also think that RFCs and ADRs are related. This may be as simple as a usability problem: When looking for RFCs or ADRs, one may see gaps in numbering. If we consider the (unique) document number as the most important attribute, then we could call documents like 002rfc and 003adr, meaning that "rfc" and "adr" are just tags of the envisioned document content.

Reply:
They can be related, sure. But that doesn't mean ADRs are "evolved" RFC. IMO, reading on the intent and traditions around either makes it immediately clear that they have been and should be distinct documents, with different content and purposes. See, for instance:

- https://news.ycombinator.com/item?id=26057068
- https://engineering.atspotify.com/2020/04/when-should-i-write-an-architecture-decision-record/
- https://medium.com/@klaus.haeuptle/shaping-architecture-decisions-from-request-for-comments-rfc-to-documenting-architecture-11dad9877342

> If we consider the (unique) document number to be the most important attribute

Why would we consider this? The document number is essentially arbitrary afaik. "the envisioned document content" seems much more important than the order in which documents were drafted imo.